src/coreclr/jit/llvm.cpp

```diff
@@ -2655,7 +2753,38 @@ void Llvm::lowerToShadowStack()
         _currentRange = &LIR::AsRange(_currentBlock);
         for (GenTree* node : CurrentRange())
         {
-            if (node->OperIs(GT_STORE_LCL_VAR, GT_LCL_VAR, GT_LCL_FLD, GT_LCL_VAR_ADDR, GT_LCL_FLD_ADDR))
+            if (node->OperIs(GT_STORE_LCL_VAR, GT_LCL_VAR, GT_LCL_VAR_ADDR))
```

Comment:
This set should include all local nodes (`OperIsLocal() || OperIsLocalAddr()`), field locals can be referred to using `[STORE_]LCL_FLD[_ADDR]` too.

Also, would be nice to outline this into its own method.

Reply:
Right, included in https://github.com/dotnet/runtimelab/pull/2007/commits/df05ac468ac2df80292e0cd739f1d2dfe09dd27f